src/main/java/core/basesyntax/service/TransactionService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface TransactionService {
+    FruitTransaction createTransaction(String s);
```

Comment:
Consider renaming the parameter 's' to a more descriptive name, such as 'transactionString'.